src/naksu/host/cpucores_linux.go

```diff
@@ -0,0 +1,36 @@
+package host
+
+import (
+	"fmt"
+	"io/ioutil"
+	"regexp"
+	"strconv"
+
+	"naksu/log"
+)
+
+// GetCPUCoreCount returns number of CPU cores
+func GetCPUCoreCount() (int, error) {
+	cpuinfoFileContentBytearr, err := ioutil.ReadFile("/proc/cpuinfo")
+
+	if err != nil {
+		log.Error("Could not open /proc/cpuinfo to detect number of CPU cores: %v", err)
+		return 0, err
+	}
+
+	cpuinfoFileContent := string(cpuinfoFileContentBytearr)
+
+	re := regexp.MustCompile(`cpu cores\s+\: (\d+)`)
```

Comment:
```suggestion
	re := regexp.MustCompile(`cpu cores\s+: (\d+)`)
```

kaksoispistettä ei tartte escapettaa, sillä ei yleensä ole erityismerkitystä regexpeissä.